ux.symfony.com/.gitignore

```diff
@@ -16,3 +16,6 @@
 /public/assets/
 /assets/vendor/
 ###< symfony/asset-mapper ###
+
+/public/site.manifest
+/public/sw.js
```

Comment:
These lines can be generated by a Flex recipe for your bundle. And maybe even the files as well unless the generation command is required ?

Reply:
Hi,

My mistake. I put those 2 lines in the early phase of the bundle development, but they are  not require anymore.
They are not generated unless asked (`symfony console asset-map:compile`).

Reply:
Removed in the last PR update